news/2020-02-03-Micro_Profile_OpenTracing_Comes_To_WildFly.adoc

```diff
@@ -0,0 +1,123 @@
+= Micro Profile OpenTracing comes to WildFly =
```

Comment:
Please update everything to remove any space between Micro and Profile. It seems to be a single camelcase word 'MicroProfile'.

Reply:
And it would be better to prefix it with Eclipse if not referring to a specific implementation. So "Micro Profile" should become "Eclipse MicroProfile" while "WildFly MicroProfile" or "SmallRye MicroProfile" is fine.